test/utils/googleMock.js

```diff
@@ -4,7 +4,7 @@ const fs = require('fs')
 const path = require('path')
 
 const {google} = require('googleapis')
-const datastore = require('@google-cloud/datastore')
+const { Datastore } = require('@google-cloud/datastore')
```

Comment:
```suggestion
const {Datastore} = require('@google-cloud/datastore')
```

According to the linter :)